src/views/tickets/orders-rsvp.php

```diff
@@ -47,13 +47,15 @@
 				?>
 			</p>
 			<?php
+			if ( ( class_exists( 'Tribe__Tickets_Plus__Attendees_List' ) ) && ! Tribe__Tickets_Plus__Attendees_List::is_hidden_on( get_the_ID() ) ) {
```

Comment:
I'd like to understand the reason for this conditional a little more. It seems to me like `event_tickets_user_details_rsvp` should fire regardless; logic like this ought to be inside the relevant callbacks.